.github/workflows/integration-per-language.yml

```diff
@@ -0,0 +1,556 @@
+        with:
+          name: draft-binary
+      - run: chmod +x ./draft
+      - run: mkdir ./langtest
+      - uses: actions/checkout@v3
+        with:
+          repository: ${{ inputs.repo }}
+          path: ./langtest
+      - name: Execute Dry Run with config file
+        run: |
+          mkdir -p test/temp
+          ./draft --dry-run --dry-run-file test/temp/dry-run.json \
+          create -c ./test/integration/${{inputs.language}}/helm.yaml \
+          -d ./langtest/ --skip-file-detection
+      - name: Validate JSON
+        run: |
+          npm install -g ajv-cli@5.0.0
+          ajv validate -s test/dry_run_schema.json -d test/temp/dry-run.json
+      - name: Execute Dry Run with variables passed through flag
+        run: |
+          mkdir -p test/temp
+          ./draft --dry-run --dry-run-file test/temp/dry-run.json \
+          create \
+          -d ./langtest/ \
+          -l ${{inputs.language}} \
+          --skip-file-detection \
+          --deploy-type helm \
+          --variable PORT=8080 \
+          --variable APPNAME=testingCreateCommand \
+          --variable VERSION=1.11 \
+          --variable BUILDERVERSION=1.11 \
+          --variable SERVICEPORT=8080 \
+          --variable NAMESPACE=testNamespace \
+          --variable IMAGENAME=testImage \
+          --variable IMAGETAG=latest \
+          --variable ENTRYPOINT=myapp.py
+      - name: Validate JSON
+        run: |
+          npm install -g ajv-cli@5.0.0
+          ajv validate -s test/dry_run_schema.json -d test/temp/dry-run.json
+  helm-create-update:
+    runs-on: ubuntu-latest
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5001:5000
+    needs: helm-dry-run
+    env:
+      imagename: registry:5001/testapp
+      serviceport: 80
+      ingress_test_args: "-a webapp_routing --variable ingress-tls-cert-keyvault-uri=test.cert.keyvault.uri --variable ingress-use-osm-mtls=true --variable ingress-host=host1"
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4.1.8
+        with:
+          name: draft-binary
+      - run: chmod +x ./draft
+      - run: mkdir ./langtest
+      - uses: actions/checkout@v3
+        with:
+          repository: ${{ inputs.repo }}
+          path: ./langtest
+      - run: |
+          rm -rf ./langtest/manifests
+          rm -f ./langtest/Dockerfile ./langtest/.dockerignore
+          rm -rf ./langtest/charts
+          rm -rf ./langtest/base
+          rm -rf ./langtest/overlays
+      - run: ./draft -v create -c ./test/integration/${{inputs.language}}/helm.yaml -d ./langtest/ --skip-file-detection
+      - name: start minikube
+        id: minikube
+        uses: medyagh/setup-minikube@master
+        with:
+          insecure-registry: "host.minikube.internal:5001,10.0.0.0/24"
+      # Runs Helm to create manifest files
+      - name: Bake deployment
+        uses: azure/k8s-bake@v3.0.1
```

Comment:
☕️ do we want to pin this to a hash?

Reply:
we can use semver when we migrate thesee to [immutable actions](https://github.com/features/preview/immutable-actions)